pkg/utils/logs/logs.go

```diff
@@ -11,72 +11,165 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-
 package logs
 
 import (
-	"flag"
 	"fmt"
 	"log"
 	"os"
-	"os/signal"
-	"syscall"
+	"os/user"
+	"path/filepath"
+	"strconv"
+	"strings"
 	"time"
 
-	"github.com/golang/glog"
-	"github.com/opensds/opensds/pkg/utils"
+	"github.com/go-ini/ini"
+	"github.com/sirupsen/logrus"
+	"gopkg.in/natefinch/lumberjack.v2"
 )
 
-const DefaultLogDir = "/var/log/opensds"
+type LogFormatter struct {
+	TimestampFormat string
+	LogFormat       string
+}
 
-// flushDaemon periodically flushes the log file buffers.
-func flushDaemon(period time.Duration) {
-	for range time.NewTicker(period).C {
-		glog.Flush()
-	}
+const (
+	debugLevel             = "debug"
+	infoLevel              = "info"
+	warnLevel              = "warn"
+	errorLevel             = "error"
+	path                   = "path"
+	level                  = "level"
+	format                 = "format"
+	configFileName         = "/etc/opensds/opensds.conf"
```

Comment:
please use OpensdsConfigPath from the file of pkg/util/constants.go instead of configFileName

Reply:
Done